starfish/core/imagestack/imagestack.py

```diff
@@ -310,7 +311,7 @@ def from_numpy(
         index_labels : Optional[Mapping[Axes, Sequence[int]]]
             Mapping from axes (r, ch, z) to their labels.  If this is not provided, then the axes
             will be labeled from 0..(n-1), where n=the size of the axes.
-        coordinates : Optional[xr.DataArray]
+        coordinates : Optional[DataArrayCoordinates]
             DataArray indexed by r, ch, z, with xmin, xmax, ymin, ymax, zmin, zmax as columns.  If
```

Comment:
This needs to be updated as well, presumably.

Reply:
contract is here. actually it's not even up to date. :/

Reply:
yea this is def not true anymore. it should just be DataArrayCoordinates object